gamemodes/terrortown/gamemode/shared/hud_elements/base_elements/pure_skin_element.lua

```diff
@@ -23,11 +23,11 @@ if CLIENT then
 		self:DrawLines(x, y, w, h)
 
 		-- draw text
-		self:ShadowedText(t or "", "PureSkinBar", x + 14, y + 5, COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_LEFT)
+		self:ShadowedText(t or "", "PureSkinBar", x + 14, y + 1, COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_LEFT)
 	end
 
 	function HUDELEMENT:ShadowedText(text, font, x, y, color, xalign, yalign)
-		draw.SimpleText(text, font, x + 1, y + 1, COLOR_BLACK, xalign, yalign)
+		draw.SimpleText(text, font, x + 1, y + 1, Color(0, 0, 0, 220), xalign, yalign)
```

Comment:
Localize this color to save performance. It‘s important in this function (draw fn!)